app/models/pageflow/linkmap_page/generated_image_file.rb

```diff
@@ -0,0 +1,33 @@
+module Pageflow
+  module LinkmapPage
+    class GeneratedImageFile < ActiveRecord::Base
+      self.abstract_class = true
+
+      include Pageflow::ReusableFile
+
+
+      belongs_to :source_image_file, class_name: 'Pageflow::ImageFile'
+
+      GEOMETRY = '1920x1080^'.freeze
+
+      Pageflow.config.file_processing_state_machine.call(ProcessSourceImageFileJob)
```

Comment:
This is a left over from a previous attempt, I guess? I'd suggest we just leave the concrete state machine in place.